ping.go

```diff
@@ -647,6 +653,9 @@ func (p *Pinger) recvICMP(
 			var err error
 			n, ttl, _, err = conn.ReadFrom(bytes)
 			if err != nil {
+				if handler := p.OnRecvError; handler != nil {
+					handler(err)
```

Comment:
Hmm, I wonder why in some cases we create a new `handler` var. But in others we do it like this:

```suggestion
				if p.OnRecvError != nil {
					p.OnRecvError(err)
```

@metalmatze Any opinion on what style we should use?

Reply:
Bump, any consensus on this? If we go with the suggestion above will that suffice for now? ie. use this pattern:

```
if p.OnRecvError != nil {
    p.OnRecvError(err)
}
```

Reply:
Yes, let's get rid of the `handler := ...` pattern.

Reply:
Updated, thanks for the input!